### PR TITLE
Additional info added for 'Enable developer tools and features' checkbox in user preferences

### LINKED ISSUE #5370: The 'Enable developer tools and features' checkbox in user preferences does not say what it does.
The 'Enable developer tools and features' checkbox in user preferences does not say what it does.

We add some more information to make it clear what the developer setting actually controls.

image

Suggest:

- Move the 'Advanced' section to over to the left, about the YAML editor settings.
- Under the checkbox for 'Enable developer tools' - add a line of text which describes what it does - e.g. 'Enables the "View in API" feature

### PR DESCRIPTION
**Pull request by neillsom**
This PR fixes #5370 by adding additional info when selecting 'enable developer tools & features' in user preferences. 

A tooltip has been added explaining the preference changes
Minor css changes have also been made to provide a better user experience

### CODE REVIEW

**Comment by neillsom** [comment]
@richard-cox PR conflicts resolved

File: shell/pages/prefs.vue [hunk 1]
```diff
@@ -201,17 +201,21 @@ export default {
       </div>
     </div>
 
+    <hr />
+    <div class="row">
+      <div class="col span-3 prefs-advanced">
```
Comment (richard-cox) [inline shell/pages/prefs.vue:206]: `span-3` causes some extreme shrinkage and can be removed

image

**Review by richard-cox — CHANGES_REQUESTED** [review]
One minor comment, otherwise LGTM

File: shell/assets/translations/en-us.yaml [hunk 2]
```diff
@@ -3551,6 +3551,7 @@ prefs:
     emacs: 'Emacs'
     vim: 'Vim'
   advanced: Advanced
+  advancedTooltip: Enables 'View in API' option, enables a 'none' locale option, <br />enables shortcuts to toggle 'none' locale (shift+l) and toggle light or dark theme (shift+t)
```
Comment (nwmac) [inline shell/assets/translations/en-us.yaml:3554]: Can we remove the information about the none locale - this is internal developer that users don't need to know. We should probably call out the 'View in API' as a specific setting at some point.

**Review by nwmac — CHANGES_REQUESTED** [review]

File: shell/assets/translations/en-us.yaml (hunk 2 above)
Comment (richard-cox) [reply shell/assets/translations/en-us.yaml]: See https://github.com/rancher/dashboard/pull/6229. `advancedTooltip` text needs updating

**Comment by nwmac** [comment]
Closed in favour of https://github.com/rancher/dashboard/pull/6276

**Pull request closed without merging**